### DISCUSSION TITLE
Upload Backups to an NFS/CIFS share

### DISCUSSION THREAD

**Discussion by dynek** [zwave-js/zwave-js-ui#3650] [Q&A]
Hello,

I'm unable to find the information anywhere. Is it possible to backup nodes+NVMe to a share, either defined in zwave-js-ui itself or using the one defined at the system level (Settings->System->Storage).

Thank you

**Comment by robertsLando** [comment]
Nope that's not possible, them are stored locally on `store` folder (that can be configured using env var). I suggest to setup a script or else that does the upload/sync to a remote folder if needed

**Comment by dynek** [comment]
Thank you.

Now I have to find out how to do it, because /data/store/ isn't exposed outside of the `zwavejs2mqtt` container.

**Reply by robertsLando** [reply]
> because /data/store/ isn't exposed outside of the zwavejs2mqtt container.

It is if you used local binding instead iof a docker volume :) Check docs: https://zwave-js.github.io/zwave-js-ui/#/getting-started/docker?id=run-using-a-local-folder

**Reply by dynek** [reply]
Did not choose, I flashed an sdcard with HomeAssistant, and everything is container based without any such setting I can see.
I just installed the add-on as explained in the doc (to replace zwave-js).

Let me see if something can be done.

**Reply by robertsLando** [reply]
@kpine Maybe can help you

**Reply by kpine** [reply]
You can create CIFS volume mounts in Docker.

If you are using HAOS, sorry, not possible to configure anything. Use HA backups and offload to network storage. HAOS supports CIFS shares.

https://www.home-assistant.io/common-tasks/os/#network-storage

**Reply by dynek** [reply]
I already did create a CIFS network storage but only HA backup can be made there, missing nodes + NVM backups from Z-Wave JS UI :-(

So if I get it right, with this setup there's no chance to backup directly to the CIFS mount point or copy them using an automation ?

**Reply by dynek** [reply]
Oh well, I should be able to something using `docker cp`.
It works from SSH, let me give it a try from an automation

**Reply by dynek** [reply]
Got it working, kind of ugly but basically SSH from HomeAssistant's container to the host (using keys to avoid password auth) then `docker cp`.

**Reply by kpine** [reply]
> I already did create a CIFS network storage but only HA backup can be made there, missing nodes + NVM backups from Z-Wave JS UI :-(

The **add-on backup** (not HA backup) includes everything in the store directory.

**Reply by dynek** [reply]
Seems I spent quite a bit of time for nothing on the matter 😂 

Thanks for the heads-up @kpine
